main.go

```diff
@@ -306,17 +307,18 @@ func main() {
 	}
 
 	// Update cordova and ionic version
-	packageManager := detectJsPackageManager(workDir)
+	packageManager := jspackage.DetectManager(workDir)
+	log.TPrintf("Js package manager used: %s", packageManager)
```

Comment:
does it worth here to use `TPrintf ` it adds the current time prefix for the message. We usually use this logging before and after command runs to highlight the command's runtime.